src/LanguageServer/Protocol/Extensions/Extensions.cs

```diff
@@ -243,5 +243,20 @@ private static bool TryGetVSCompletionListSetting(ClientCapabilities clientCapab
 
             return true;
         }
+
+        public static int CompareTo(this Position p1, Position p2)
```

Comment:
>CompareTo [](http://example.com/codeflow?start=26&length=9)

I'm sure there is a similar method implemented somewhere in Roslyn, but I couldn't find it